docs/release_notes/8.4.md

```diff
@@ -4,9 +4,34 @@ title:  Rocky Linux 8.4 Release Notes
 
 # Rocky Linux 8.4
 
-The entire Rocky Linux team is excited to let you know about the first full release of Rocky Linux, Rocky Linux 8.4! If you are new to Rocky Linux, it's great to have you here and we look forward to a long and happy relationship.
+We are pleased to announce the General Availability of Rocky Linux 8.4 (Green Obsidian).
 
-Rocky Linux is a community enterprise operating system designed to be 100% bug-for-bug compatible with America's top enterprise Linux distribution. It is under intensive development by the community. As such, the release notes reflect the changes to the upstream packages and to those included in the original Rocky Linux 8.3 release candidate.
+Rocky Linux is a community enterprise operating system designed to be 100% bug-for-bug compatible with  Red Hat Enterprise Linux 8.4. Since this is the first Release of Rocky Linux, the release notes below reflect only changes in upstream functionality between point releases.
+
+## Thank You
+
+The entire Rocky Linux team wishes to thank everyone in the community that has supported the project since its inception in a comment on a blog post. From feedback and well wishes, to bug reports, donations, and sponsorship, our community has come through again and again. The countless users who have installed and tested Rocky Linux release candidates on their hardware, making sure their software will work with Rocky are owed a debt of gratitude not only from us, but from the entire Enterprise Linux community.
+
+You have made Rocky Linux what it is today, and as we go forward from this release, we hope the community continues to grow and be able to support more like-minded projects to reach their goals as well.
+
+## Supported Upgrades
+
+> :exclamation: Migration to Rocky Linux 8.4 is **not** supported from Rocky Linux 8.3 RC1, Rocky Linux 8.4 RC1, or any other release candidates.
+
+### Conversion Tool Available
+
+The community has created the [migrate2rocky](https://github.com/rocky-linux/rocky-tools/tree/main/migrate2rocky) tool to aid in the conversion to Rocky Linux 8.4 from other Enterprise Linux systems.
+
+This tool has been tested and is generally known to work, however use of it is **at your own risk**.
+
+Community members have successfully migrated test systems to Rocky Linux from:
+
+* Alma Linux (8.4)
+* CentOS Linux (8.4)
+* Red Hat Enterprise Linux (8.4)
+* Oracle Linux (8.4)
+
```

Comment:
Also Springdale Linux

Reply:
Update: Oracle 8 works, but it fails to remove a conflicting package (redhat-release) which can cause issues down the road if it's not removed from the db, specifically it might cause problems with dnf update if rocky-release needs an update.  I will be looking for a solution for this, but a temporary workaround is to run this command after the migration:
`rpm -e --justdb redhat-release`

Reply:
The Oracle Linux issue has been fixed with PR #36

Reply:
Alma Linux 8.4 and Springdale Linux 8.4 are all tested and known to work as well.